### ISSUE TITLE
Picture / Logo Upload

### ISSUE THREAD

**Issue by hallerdavid** [JhumanJ/OpnForm#570]
**Describe the issue**
I cannot upload Logo or pictures to form, the popup let me choose the file but after clicking upload it just closes again.
**Deployment Environment**

-   OpnForm Version: Docker-Compose tells me it uses the latest - where can i find the version?
-   Hosting Platform: VPS Server on Netcup infrastructure
-   OS: Debian 12 Bookworm

**Deployment Method**

-   [ X] Docker-Compose
-   [ ] Manual installation
-   [ ] Other (please specify)

**Steps Taken**
Describe the steps you've taken to deploy the application:

1. Installed the script from Github
2. customized .env and /client/.env
3. docker compose up -d

**Error Messages**
![image](https://github.com/user-attachments/assets/9a6c10e1-3f1a-4252-8ac8-93c33af71e1a)
in console i see this, there is no error in the UI.

**Configuration Files**    
**.env:**                              
```
APP_NAME="OpnForm"
APP_ENV=local
APP_KEY=KEY
APP_DEBUG=false
APP_URL=URL TO APP
LOG_CHANNEL=errorlog
LOG_LEVEL=debug

FILESYSTEM_DRIVER=local

BROADCAST_CONNECTION=log
CACHE_STORE=redis
QUEUE_CONNECTION=redis
SESSION_DRIVER=redis
SESSION_LIFETIME=120

MAIL_MAILER=log
MAIL_HOST=SERVER
MAIL_PORT=465
MAIL_USERNAME="MAIL"
MAIL_PASSWORD="PASSWORD"
MAIL_ENCRYPTION=TLS
MAIL_FROM_ADDRESS="MAIL"
MAIL_FROM_NAME="NAME"

AWS_ACCESS_KEY_ID=
AWS_SECRET_ACCESS_KEY=
AWS_DEFAULT_REGION=us-east-1
AWS_BUCKET=

PUSHER_APP_ID=
PUSHER_APP_KEY=
PUSHER_APP_SECRET=
PUSHER_APP_CLUSTER=mt1

MIX_PUSHER_APP_KEY="${PUSHER_APP_KEY}"
MIX_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"

JWT_TTL=1440
JWT_SECRET=SECRET

MUX_WORKSPACE_ID=
MUX_API_TOKEN=

OPEN_AI_API_KEY=
SELF_HOSTED=true
```

**/client/.env:**
```
NUXT_LOG_LEVEL=
NUXT_PUBLIC_APP_URL=URL TO FORMS
NUXT_PUBLIC_API_BASE=URL TO FORMS/api
NUXT_PRIVATE_API_BASE=http://ingress/api
NUXT_PUBLIC_AI_FEATURES_ENABLED=false
NUXT_PUBLIC_AMPLITUDE_CODE=
NUXT_PUBLIC_CRISP_WEBSITE_ID=
NUXT_PUBLIC_CUSTOM_DOMAINS_ENABLED=
NUXT_PUBLIC_ENV=local
NUXT_PUBLIC_GOOGLE_ANALYTICS_CODE=
NUXT_PUBLIC_H_CAPTCHA_SITE_KEY=
NUXT_PUBLIC_PAID_PLANS_ENABLED=
NUXT_PUBLIC_S3_ENABLED=false
NUXT_API_SECRET=SECRET
```
 

**Logs**
api/storage/logs is not there, i have no folder "api" in my "OpnForms" folder? I created the folder API for set DEBUG to true (like above)
-   Nuxt logs in the client docker logs --> where can i find this?

**Comment by hallerdavid** [comment]
![image](https://github.com/user-attachments/assets/a09d8c8e-86d6-4d30-ad50-670573d1123f)
also i have this in my console

**Comment by hallerdavid** [comment]
Fixed with #565 !
(wrong script / wrong .env files)
